tools/flash_rx_via_ap_passthru3.py

```diff
@@ -285,9 +285,14 @@ def mlrs_flash_firmware_file(uart, baud, file, full_erase_flag):
     arg += ' -w "' + file + '"'
     arg += ' -v'
     arg += ' -g'
-    res = os.system(os.path.join('zmodules','STM32CubeProgrammer','bin','STM32_Programmer_CLI.exe') + ' ' + arg)
+    st_path = os.path.join('zmodules','STM32CubeProgrammer','bin','STM32_Programmer_CLI.exe')
+    if os.name == 'posix': # Assume local install
+        st_path = os.path.join("~",'STMicroelectronics','STM32Cube','STM32CubeProgrammer','bin','STM32_Programmer_CLI')
+
+    res = os.system(st_path + ' ' + arg)
     if res > 0:
         print('ERROR: Flashing firmware failed!')
+        print('Please flash manually and then power cycle')
```

Comment:
what is this error message supposed to tell?
if it failed why would flashing "manually" work?

Reply:
Because on Linux it may have failed due to not finding the tool in the expected location.

Reply:
On windows, I suppose a manual use of STM32CubeProgrammer may also fix an issue like write protection, etc.

Reply:
frankly, sounds speculative ... IMHO better just to tell what sort of issue occured instead of trying a suggeestion which may not help
this reads as ifthe user is supposed to know what to do and that it will be the unambigious resolution to the occuring issue. I think none is necessarily correct.
IMHO.

Reply:
ah, So rather something like "ERROR: Flashing firmware failed. Receiver is however still in systemboot and trying to manually flash might work."